osu.Desktop/osu.nuspec

```diff
@@ -11,7 +11,7 @@
         <requireLicenseAcceptance>false</requireLicenseAcceptance>

```

Comment:
The `iconUrl` line above this has a really outdated logo. Not sure if it's shown anywhere now, but here's the doc for more investigation: https://learn.microsoft.com/en-us/nuget/reference/nuspec#iconurl